events/events-intro.md

```diff
@@ -0,0 +1,67 @@
+# Introduction to event-based transactions with openDS
+
+
+![Interaction paths between DiSSCo technical infrastructure, CMSes and additional research or application systems](https://github.com/DiSSCo/openDS/blob/jbstatgen_events/events/FullPath_Detail_20221102b.drawio.png "FullPath_Detail_png")
+
+**Figure 1**: Graphical summary of the interaction between CMSes, DOA and additional external systems for research and application, e.g. analytical environments or national reporting for biodiversity monitoring.
+
+This introduction describes the use case of a local Content Management System (CMS) in use by a natural science institution or scientific collection interacting with an online infrastructure, in this case the technical Research Infrastructure of the European [Distributed System of Scientific Collections](https://www.dissco.eu/) (DiSSCo RI). The DiSSCo RI will be implemented following the concept of a [Digital Object Architecture](https://www.dona.net/digitalobjectarchitecture) (DOA) and will be part of a globally integrated DOA.
+
+## Basic CMS - DiSSCo DOA interaction
+
+The interface between a CMS and the DiSSCo Digital Object Architecture (DOA), or DiSSCo Research Infrastructure (RI), involves at a minimum **two agents** and an **abstraction layer** that connects the information flow between the two agents through automated event-based transactions including at least one mapping.
+
+### Agents
+
+In the interaction, the DiSSCo DOA and each CMS endpoint are recognized as **agents**. Thereby, each agent is uniquely identified by a PID (persistent, globally unique and resolvable identifier). In addition to these two types of agents, it is planned that automated machine processes externally originating in, for example, research environments will be able to initiate and run transactions as independent agents. 
+
+In a first development step, agents involved in transactions are only recognized at the organization level, that is, recorded for a transaction on the side of CMS partners are PIDs of institutions (e.g. RORs) or CMS endpoints. It is planned that in the future also agents active within the CMS, for example, individual staff and scientists using and contributing to the data within the CMS can be associated with transactions through their specific PIDs (identified by e.g. ORCIDs), and thus become visible not only within the CMS, but also to the DOA. 
```

Comment:
We can indicate this but need to be carefully as I think this will be a while before we can get there. Additionally I haven't seen room in the data standards where it can be specified who changed the digital Specimen.

Reply:
"One option for expanding the interaction model in future development steps is that also agents active within the CMS, ..."

Would this be a more careful way to phrase it? I find it important that readers and the community learn about this possibility and that it is on the radar. For many such functionality is very important and it will be good to show that the functionality that they wish for is part of the plan.

Mmh, a) the information should be part of a module equivalent eg. to `ltc:RecordLevel`. It might be introduced with the event-based transaction model. b) I thought that PROV and OA provide options for associating attribution to a change.